src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -158,5 +159,23 @@ trait ThurloeService extends HttpService {
     }
   }
 
-  val keyValuePairRoutes = getRoute ~ getAllRoute ~ queryRoute ~ setRoute ~ deleteRoute
+  val getStatus = path("status") {
```

Comment:
@MatthewBemis Should I make a separate StatusService trait and put this there? I just realized that's what we're doing with the notification stuff..